tests/rptest/tests/transaction_kafka_api_test.py

```diff
@@ -29,7 +29,8 @@ def __init__(self, test_context):
                                  "abort_timed_out_transactions_interval_ms":
                                  10000000,
                                  'enable_leader_balancer': False
-                             })
+                             },
+                             log_level="trace")
```

Comment:
why this? I think cdt default is trace with trimming on success?

Reply:
cdt isn't the only place when we run ducktape and redpanda.py's DEFAULT_LOG_LEVEL is info

Reply:
ok, what specifically is this for? Out of curiosity.

Reply:
my mistake, I was under impression that pr ci-runs use default log level (info) but I double checked and it's trace. I'll roll back this change, thanks!